### ISSUE TITLE
Allow left click to continuously apply certain brushes

### ISSUE THREAD

**Issue by delzhand** [delzhand/isocon#343]
When mouse button is held down:
Add Height - apply to each column the mouse cursor hits, but only once
Remove Height - same as add height, only apply to each column once
Change Shape - apply to each tile hit but without the rotate effect (see how this feels in practice, this may not be desirable)
Paint/Texture - apply to each tile hit

**Comment by McPalm** [comment]
I can already tell, that that the remove height will have some funny results. Since you select with a raycast on the blocks, and then the block will disappear, meaning in the next frame you will remove height on the block behind.

This could be easily solved by having a temporary plane when editing elevations and raycast off that. But that leads to the need of raycast filters for the cursor, which it did not have last week, but I can look into adding that.

For the change shape, I suggest having some kind of geometry deteciton that determines the most sensible rotation, stairs generally want to connect different elevations, rounded corners generally want to be on the corner, etc.

**Comment by McPalm** [comment]
Due to how Change Shap  work right now, it ended up spinning them around, very fun! But not very useful. Styling could use some improvements overall I think, and should be moved to a separate issue.
